File: examples/jvm/src/main/java/example/Hello.java
package example;

import java.io.FileOutputStream;
import jp.co.field_works.field_reports.*;

public class Hello 
{
    public static void main(String[] args)
    {
        String param = "{\n" +
"   \"template\": {\"paper\": \"A4\"},\n" +
"   \"context\": {\n" +
"       \"hello\": {\n" +
"           \"new\": \"Tx\",\n" +
"           \"value\": \"Hello, World!\",\n" +
"           \"rect\": [100, 700, 400, 750]\n" +
"       }\n" +
"   }\n" +
"}\n";
        Proxy reports = Bridge.createProxy(null);
        try {
            FileOutputStream fos = new FileOutputStream(args[0]);
            byte[] pdf = reports.render(param);
            fos.write(pdf);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: examples/php/hello.php
<?php
require('vendor/autoload.php');

$param = [
    "template" => ["paper" => "A4"],
    "context" => [
        "hello" => [
            "new" => "Tx",
            "value" => "Hello, World!",
            "rect" => [100, 700, 400, 750]
        ]
    ]
];

$reports = FieldReports\Bridge::create_proxy();
$fp = fopen($argv[1], 'w');
fwrite($fp, $reports->render($param));
?>
